<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>generalScriptEditor</class>
 <widget class="QMainWindow" name="generalScriptEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>427</width>
    <height>294</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SQL Script Editor</string>
  </property>
  <widget class="QWidget" name="layout_text">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_filename">
      <property name="text">
       <string>(Unsaved)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="text_editor">
      <property name="plainText">
       <string/>
      </property>
      <property name="tabStopWidth">
       <number>4</number>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="button_executeScript">
        <property name="text">
         <string>Execute Script</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>427</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_new"/>
    <addaction name="action_open"/>
    <addaction name="action_save"/>
    <addaction name="action_saveAs"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="action_new">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="action_open">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="action_save">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="action_saveAs">
   <property name="text">
    <string>Save As...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
